src/metax_api/api/rest/base/api_schemas/datacatalog.json

```diff
@@ -59,7 +59,14 @@
                     "description":"Service name who last modified this resource.",
                     "type":"string",
                     "readonly": true
-                }
+                },
+                "date_removed":{
+                    "title":"Date removed",
+                    "description":"Date of removal of the resource. Can't be updated once set",
+                    "type":"string",
+                    "format":"date-time",
+                    "readonly":true
+                  }
```

Comment:
Looks like field ``removed`` has been missing from this file too. Good time to add it.

Reply:
Actually that field looks like it's missing from all three api_schemas. I'll add it to them.